### ISSUE TITLE
[Feature Request] Add a way to listen to every song in the library with modular tabs

### ISSUE THREAD

**Issue by andreasmolnardev** [avdept/JellyBoxPlayer#91]
Currently, the only supported ways to play music are via Albums, Artists and Playlists. However neither do I need Albums nor Artists. Could there be a way to implement a modular approch regarding the tab items at the top where a list of all songs are listed?

**Comment by avdept** [comment]
> Currently, the only supported ways to play music are via Albums, Artists and Playlists. However neither do I need Albums nor Artists. Could there be a way to implement a modular approch regarding the tab items at the top where a list of all songs are listed?

So just list all songs? 

Just checked - Jellyfin supports this mode and has API for list all songs. I'll see what can I do, since I'd need different UI for showing list of songs

**Comment by andreasmolnardev** [comment]
Yeah, basically. Curious to see whats posibble! Thanks to this feature or a similiar one, even if it's just being able to put an entire library into one playlist, i'll definitly be on board and able to use Jellybox on a regular basis. I really appreciate your efforts!

**Comment by avdept** [comment]
@andreasmolnardev this was implemented. Now app has a raw list of songs. In fact I use it mostly myself too in random order. Sometimes finding a really nice songs
